scripts/hypothesistooling.py

```diff
@@ -149,12 +150,18 @@ def create_tag_and_push():
         'git push ssh-origin --tags',
         'git push ssh-origin HEAD:master',
     ]:
-        subprocess.check_call([
-            'ssh-agent', 'sh', '-c',
-            'chmod 0600 deploy_key && ' +
-            'ssh-add deploy_key && ' +
-            command
-        ])
+        for retries_left in range(2, -1, -1):
+            try:
+                subprocess.check_call([
+                    'ssh-agent', 'sh', '-c',
+                    'chmod 0600 deploy_key && ' +
+                    'ssh-add deploy_key && ' +
+                    command
+                ])
+            except subprocess.CalledProcessError:
+                if retries_left == 0:
+                    raise
+                time.sleep(5)
```

Comment:
Do we want an `else` block that skips the remaining retries if the command succeeds, or have we seen commands that report a zero exit code and still fail?